mkdocs.yml

```diff
@@ -104,6 +110,17 @@ nav:
       - Using the App: "user/app_use_cases.md"
       - Frequently Asked Questions: "user/faq.md"
       - External Interactions: "user/external_interactions.md"
+      - Data Models:
+          - DNS Zone: "models/dnszonemodel.md"
+          - DNS Record: "models/dnsrecordmodel.md"
+          - NS Record: "models/nsrecordmodel.md"
+          - A Record: "models/arecordmodel.md"
+          - AAAA Record: "models/aaaarecordmodel.md"
+          - CNAME Record: "models/cnamerecordmodel.md"
+          - MX Record: "models/mxrecordmodel.md"
+          - PTR Record: "models/ptrrecordmodel.md"
+          - SRV Record: "models/srvrecordmodel.md"
+          - TXT Record: "models/txtrecordmodel.md"
   - Administrator Guide:
```

Comment:
If we need the class attributes in every page, then sure let's add those. Otherwise I am happy to skip all those pages for now, and revisit later :)